src/Dialog/Dialog.js

```diff
@@ -141,7 +141,7 @@ function getStyles(props, context) {
       fontSize: dialog.bodyFontSize,
       color: dialog.bodyColor,
       padding: gutter,
-      paddingTop: props.title ? 0 : gutter,
+      paddingTop: 0,
```

Comment:
![screen shot 2016-04-25 at 2 43 06 pm](https://cloud.githubusercontent.com/assets/15271922/14796655/133af63e-0af4-11e6-8fe3-f4be057cf495.png)

This is what I was saying. If I don't mention a `paddingTop: 0`